src/components/KeyboardCompatibleView/hooks/useKeyboardCompatibleHeight.ts

```diff
@@ -1,12 +1,23 @@
-import { useCallback, useEffect, useRef, useState } from 'react';
-import { Dimensions, Keyboard, Platform, StatusBar } from 'react-native';
+import { RefObject, useCallback, useEffect, useRef, useState } from 'react';
+import {
+  Dimensions,
```

Comment:
should we be switching to useWindowDimensions since it's now the preferred way?
https://reactnative.dev/docs/dimensions

Reply:
yeah